core/cafe_api_v1.go

```diff
@@ -631,3 +631,18 @@ func (c *cafeApi) search(g *gin.Context) {
 		return true
 	})
 }
+
+// sendError sends the error to the gin context
+func sendError(g *gin.Context, err error, statusCode int) {
+	g.String(statusCode, err.Error())
+}
+
+// pbJSON responds with a JSON rendered protobuf message
+func pbJSON(g *gin.Context, status int, msg proto.Message) {
+	str, err := pbMarshaler.MarshalToString(msg)
+	if err != nil {
+		sendError(g, err, http.StatusBadRequest)
+		return
+	}
+	g.Data(status, "application/json", []byte(str))
+}
```

Comment:
cafe_api will obv need to be it's own mod soon too, for now i just made this work as is